src/main/webapp/app/iris/about-iris/about-iris.component.ts

```diff
@@ -1,11 +1,17 @@
-import { Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
 import { faRobot } from '@fortawesome/free-solid-svg-icons';
-import { IrisLogoSize } from '../iris-logo/iris-logo.component';
+import { IrisLogoComponent, IrisLogoSize } from '../iris-logo/iris-logo.component';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { IrisModule } from 'app/iris/iris.module';
 
 @Component({
     selector: 'jhi-about-iris',
     templateUrl: './about-iris.component.html',
     styleUrls: ['about-iris.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    standalone: true,
+    imports: [TranslateDirective, ArtemisSharedModule, IrisModule, IrisLogoComponent],
```

Comment:
I'm not sure, but do you need the complete ArtemisShareModule here? (For standalone it is much better to only import the things you really need for a component)

Reply:
Yes, ArtemisSharedModule is needed for artemisTranslate in the html.